cider/tests/substrait_plan_files/extract/day_of_week_not_null.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
What's the different from [day_of_week_not_null](https://github.com/intel/BDTK/blob/main/cider/tests/substrait_plan_files/extract/day_of_week.json)? It would be great to add some comments either in this file or [cider/tests/functionality/DateTypeQueryTest.cpp](https://github.com/intel/BDTK/pull/146/files#diff-c6ecb6a2e11833acedb7fb05b892b304d7441a22403c94f477e9482ed5212f77)